src/BlockChain/custom/New-AzBlockchainTransactionNode.ps1

```diff
@@ -0,0 +1,148 @@
+.Outputs
+Microsoft.Azure.PowerShell.Cmdlets.Blockchain.Models.Api20180601Preview.ITransactionNode
+.Notes
+COMPLEX PARAMETER PROPERTIES
+To create the parameters described below, construct a hash table containing the appropriate properties. For information on hash tables, run Get-Help about_Hash_Tables.
+
+FIREWALLRULE <IFirewallRule[]>: Gets or sets the firewall rules.
+  [EndIPAddress <String>]: Gets or sets the end IP address of the firewall rule range.
+  [RuleName <String>]: Gets or sets the name of the firewall rules.
+  [StartIPAddress <String>]: Gets or sets the start IP address of the firewall rule range.
+.Link
+https://docs.microsoft.com/en-us/powershell/module/az.blockchain/new-azblockchaintransactionnode
+#>
+function New-AzBlockchainTransactionNode {
+[OutputType([Microsoft.Azure.PowerShell.Cmdlets.Blockchain.Models.Api20180601Preview.ITransactionNode])]
+[CmdletBinding(DefaultParameterSetName='CreateExpanded', PositionalBinding=$false, SupportsShouldProcess, ConfirmImpact='Medium')]
+[Microsoft.Azure.PowerShell.Cmdlets.Blockchain.Profile('latest-2019-04-30')]
+param(
+    [Parameter(Mandatory)]
+    [Microsoft.Azure.PowerShell.Cmdlets.Blockchain.Category('Path')]
+    [System.String]
+    # Blockchain member name.
+    ${BlockchainMemberName},
+
+    [Parameter(Mandatory)]
+    [Alias('TransactionNodeName')]
+    [Microsoft.Azure.PowerShell.Cmdlets.Blockchain.Category('Path')]
+    [System.String]
+    # Transaction node name.
+    ${Name},
+
+    [Parameter(Mandatory)]
+    [Microsoft.Azure.PowerShell.Cmdlets.Blockchain.Category('Path')]
+    [System.String]
+    # The name of the resource group that contains the resource.
+    # You can obtain this value from the Azure Resource Manager API or the portal.
+    ${ResourceGroupName},
+
+    [Parameter()]
+    [Microsoft.Azure.PowerShell.Cmdlets.Blockchain.Category('Path')]
+    [Microsoft.Azure.PowerShell.Cmdlets.Blockchain.Runtime.DefaultInfo(Script='(Get-AzContext).Subscription.Id')]
+    [System.String]
+    # Gets the subscription Id which uniquely identifies the Microsoft Azure subscription.
+    # The subscription ID is part of the URI for every service call.
+    ${SubscriptionId},
+
+    [Parameter()]
+    [Microsoft.Azure.PowerShell.Cmdlets.Blockchain.Category('Body')]
+    [Microsoft.Azure.PowerShell.Cmdlets.Blockchain.Models.Api20180601Preview.IFirewallRule[]]
+    # Gets or sets the firewall rules.
+    # To construct, see NOTES section for FIREWALLRULE properties and create a hash table.
+    ${FirewallRule},
+
+    [Parameter()]
+    [Microsoft.Azure.PowerShell.Cmdlets.Blockchain.Category('Body')]
+    [System.String]
+    # Gets or sets the transaction node location.
+    ${Location},
+
+    [Parameter()]
+    [Microsoft.Azure.PowerShell.Cmdlets.Blockchain.Category('Body')]
+    [System.Security.SecureString]
+    # Sets the transaction node dns endpoint basic auth password.
```

Comment:
Should have ValidateNotNullOrEmpty attribute, or the code below should also verify the password is not null or empty